e2e/tests/noticeOfChange_test.js

```diff
@@ -0,0 +1,26 @@
+const config = require('../config.js');
+const mandatorySubmissionFields = require('../fixtures/caseData/mandatorySubmissionFields.json');
+
+let caseId;
+
+Feature('Notice of change');
+
+Scenario('Private solicitor obtains case access through NoC', async ({I, caseListPage, noticeOfChangePage}) => {
```

Comment:
Test will fail until https://tools.hmcts.net/jira/browse/FPLA-2962 is implemented.